src/pids.c

```diff
@@ -135,9 +229,9 @@ void decode_sis(pids_t *st, uint8_t *bits)
 
             if ((strcmp(country_code, st->country_code) != 0) || (fcc_facility_id != st->fcc_facility_id))
             {
-                log_debug("Country: %s, FCC facility ID: %d", country_code, fcc_facility_id);
                 strcpy(st->country_code, country_code);
                 st->fcc_facility_id = fcc_facility_id;
+                report(st);
```

Comment:
It looks like ```report()``` is going to potentially be called many times as the structure is updated. Might it be better to have a ```changed``` variable and call ```report()``` once at the end of the function if ```changed == true```?

Reply:
I might try that out, but the gains won't be huge since there are at most two SIS messages per PIDS frame.

Reply:
The improvement turned out to be about 40%. I also added a couple extra checks to make sure that location is only reported when both latitude and longitude are ready. (They arrive in two separate messages.)